pennylane/pulse/hardware_hamiltonian.py

```diff
@@ -179,101 +179,82 @@ def circuit_local(params):
-        ]
-
-        drive_x_term = 0.5 * sum(qml.PauliX(wire) for wire in wires)
-        drive_y_term = -0.5 * sum(qml.PauliY(wire) for wire in wires)
-
-        observables = [drive_x_term, drive_y_term]
+    # We compute the `coeffs` and `observables` of the EM field
+    coeffs = [
+        amplitude_and_phase(qml.math.cos, amplitude, phase),
+        amplitude_and_phase(qml.math.sin, amplitude, phase),
+    ]
 
-    if not trivial_detuning:
-        detuning_term = -1.0 * sum(qml.PauliZ(wire) for wire in wires)
-        coeffs.append(detuning)
-        observables.append(detuning_term)
+    drive_x_term = 0.5 * sum(qml.PauliX(wire) for wire in wires)
+    drive_y_term = -0.5 * sum(qml.PauliY(wire) for wire in wires)
 
-    # We convert the pulse data into a list of ``HardwarePulse`` objects
-    pulses = [HardwarePulse(amplitude, phase, detuning, wires)]
+    observables = [drive_x_term, drive_y_term]
 
-    return HardwareHamiltonian(coeffs, observables, pulses=pulses)
+    return HardwareHamiltonian(coeffs, observables)
 
 
 class HardwareHamiltonian(ParametrizedHamiltonian):
     r"""Internal class used to keep track of the required information to translate a ``ParametrizedHamiltonian``
     into hardware.
-
     This class contains the ``coeffs`` and the ``observables`` to construct the :class:`ParametrizedHamiltonian`,
     but on top of that also contains attributes that store parameteres relevant for real hardware execution.
-
     .. warning::
-
         This class should NEVER be initialized directly! Please use the functions
         :func:`rydberg_interaction` and :func:`drive` instead.
-
     .. seealso:: :func:`rydberg_interaction`, :func:`drive`, :class:`ParametrizedHamiltonian`
-
     Args:
         coeffs (Union[float, callable]): coefficients of the Hamiltonian expression, which may be
             constants or parametrized functions. All functions passed as ``coeffs`` must have two
             arguments, the first one being the trainable parameters and the second one being time.
         observables (Iterable[Observable]): observables in the Hamiltonian expression, of same
             length as ``coeffs``
-
     Keyword Args:
-        register (list): list of coordinates (in micrometers) of each atom in the ensemble
-        pulses (list): list of ``HardwarePulse`` classes containing the information about the
-            amplitude, phase, detuning and wires of each pulse
-        interaction_coeff (float): Rydberg interaction constant in units: :math:`\text{MHz} \times \mu m^6`.
-            Defaults to :math:`862690 \text{MHz} \times \mu m^6`.
-
+        settings Union[RydbergSettings, TransmonSettings]: Dataclass containing the hardware specific settings. Default is ``None``.
+        pulses (list[HardwarePulse]): list of ``HardwarePulse`` dataclasses containing the information about the
+            amplitude, phase, drive frequency and wires of each pulse
```

Comment:
All the new lines between the different sections of the documentation of `HardwareHamiltonian` are no longer present and it's making the docstring look quite cramped.

Reply:
Oh you are right! I know the cause for this, will fix be fore merging